internal/cli/atlas/quickstart/prompt.go

```diff
@@ -48,9 +48,10 @@ func newClusterProviderQuestion() *survey.Question {
 }
 
 func newAccessListQuestion(publicIP, message string) survey.Prompt {
+	extraInfo := "  Use comma(,) to separate multiple entries. Use \"--allowAll\" to enable connection from anywhere."
```

Comment:
saying `Use \"--allowAll\"` it's not helpful here, it means I have to cancel the command and start again, or even some users will type `--allowAll` instead of an IP

Reply:
no, for the guided flow feel free to keep the "use 0.0.0.0/1" thingy, this became kind of a mess of a review mainly because of the string reuse for flag help, the original copy was perfect for the interactive/guided flow, but flawed for flag help and I think all my comments were about the help on flag